changelog/unreleased/change-deprecate-autocomplete

```diff
@@ -0,0 +1,5 @@
+Change: deprecate OcAutocomplete
```

Comment:
Deprecating isn't a change in itself, I'd vote in favor of having the deprecation as an enhancement in the changelog and then dropping all deprecated components in one of the upcoming major releases

Reply:
I agree, dropping the component will be the change IMO